src/app/components/ms-identifier/ms-identifier.component.html

```diff
@@ -0,0 +1,74 @@
+<div class="msIdentifier">
+    <evt-ms-desc-section [label]="'msIdentifier'">
+        <div content>   
+            <evt-content-viewer></evt-content-viewer>
+        </div>
+    </evt-ms-desc-section>
+    <div>
+        <evt-ms-desc-section [label]="'place'" [additionalClass]="'settlement'" [inlineLabel]="'inlineLabel'">
+            <div content>   
+                <evt-content-viewer *ngFor="let s of data?.settlement" [content]="s"></evt-content-viewer>
+                <ng-container *ngIf="data?.region.length>0">
+                    <ng-container *ngIf="data.settlement.length>0">, </ng-container>
+                    <evt-content-viewer *ngFor="let r of data.region" [content]="r"></evt-content-viewer>
+                </ng-container>
+                <ng-container *ngIf="data?.country.length>0">
+                    <ng-container *ngIf="data.settlement.length>0 || data.region.length>0">, </ng-container>
+                    <evt-content-viewer *ngFor="let c of data.country" [content]="c"></evt-content-viewer>
+                </ng-container> 
+            </div>
+        </evt-ms-desc-section>
+        <evt-ms-desc-section *ngIf="data?.repository" [label]="'repository'" [additionalClass]="'repository'" [inlineLabel]="'inlineLabel'">
+            <div content>   
+               <evt-content-viewer [content]="data.repository"></evt-content-viewer>
+            </div>
+        </evt-ms-desc-section>
+        <evt-ms-desc-section *ngIf="data?.institution" [label]="'institution'" [additionalClass]="'institution'" [inlineLabel]="'inlineLabel'">
+            <div content>   
+               <evt-content-viewer [content]="data.institution"></evt-content-viewer>
+            </div>
+        </evt-ms-desc-section>
+        <evt-ms-desc-section *ngIf="data?.collection.length>0" [label]="'collection'" [additionalClass]="'collection'" [inlineLabel]="'inlineLabel'">
+            <div content>   
+               <evt-content-viewer *ngFor="let c of data.collection" [content]="c" class="projectInfo-block"></evt-content-viewer>
+            </div>
+        </evt-ms-desc-section>
+        <evt-ms-desc-section *ngFor="let m of data?.msName | keyvalue" [label]="'msName'" [additionalClass]="'msName'" [inlineLabel]="'inlineLabel'">
+            <div content>   
+               <evt-content-viewer [content]="data.msName[m.key]"></evt-content-viewer>
+            </div>
+        </evt-ms-desc-section>
+        <evt-ms-desc-section *ngIf="data?.idno.length>0" [label]="'idno'" [additionalClass]="'idno'" [inlineLabel]="'inlineLabel'">
+            <div content>   
+               <evt-content-viewer *ngFor="let i of data.idno" [content]="i" class="projectInfo-block"></evt-content-viewer>
+            </div>
+        </evt-ms-desc-section>
+        <span *ngFor="let a of data?.altIdentifier | keyvalue" >
```

Comment:
As before, you don't need the keyvalue pipe; just do `*ngFor="let a of data?.altIdentifier"` and then use directly `a` for bindings instead of `data.altIdentifier[a.key]`